plugins/keyboard/gsd-keyboard-manager.c

```diff
@@ -496,12 +494,10 @@ get_sources_from_xkb_config (GsdKeyboardManager *manager)
                 g_variant_unref (v);
         }
 
-        init_builder_with_sources (&builder, priv->input_sources_settings);
+        if (!layouts)
```

Comment:
Nitpick: Typo s/layoyt/layout/ in the commit message “keyboard: don't always add an US layoyt”

Reply:
done